crates/cli/src/subcommands/server.rs

```diff
@@ -425,26 +432,10 @@ pub async fn exec_edit(mut config: Config, args: &ArgMatches) -> Result<(), anyh
 
 async fn exec_clear(_config: Config, args: &ArgMatches) -> Result<(), anyhow::Error> {
     let force = args.get_flag("force");
-    if std::env::var_os("STDB_PATH").map(PathBuf::from).is_none() {
-        let mut path = dirs::home_dir().unwrap_or_default();
-        path.push(".spacetime");
-        std::env::set_var("STDB_PATH", path.to_str().unwrap());
-    }
-
-    let control_node_dir = stdb_path("control_node");
-    let worker_node_dir = stdb_path("worker_node");
-    if control_node_dir.exists() || worker_node_dir.exists() {
-        if control_node_dir.exists() {
-            println!("Control node database path: {}", control_node_dir.to_str().unwrap());
-        } else {
-            println!("Control node database path: <not found>");
-        }
+    let data_dir = args.get_one::<PathBuf>("data_dir").unwrap();
```

Comment:
I think it is better if anything that is related to the `Layout` is described by constants instead of hardcode strings.

ie: `Paths::DATA_DIR`

Reply:
This is just the clap argument though, it's defined in the same file and doesn't matter anywhere else. It could be renamed to `"foo"` and it wouldn't be observable to the user at all.